doc/userdoc/guides/stimulate_the_network.rst

```diff
@@ -23,7 +23,7 @@ stimuli for connected nodes. Such data might be explicit lists of
 values, or parametric values as for instance amplitudes and
 frequencies.
 
-.. _stimulating_backends:
+.. _stimulation_backends:
```

Comment:
```suggestion
.. _sec_stimulation_backends:
```

Reply:
Following the styleguide adding the sec precursor. I couldnt find anywhere where this is referenced, but if I missed something please update the ref as well.